dipy/denoise/noise_estimate.py

```diff
@@ -295,9 +302,9 @@ def estimate_sigma(arr, disable_background_masking=False, N=0):
     (see [1]_, equation 18) with theta = 0.
     Since this function was introduced in [2]_ for T1 imaging,
     it is expected to perform ok on diffusion MRI data, but might oversmooth
-    some regions and leave others un-denoised for spatially varying noise profiles.
-    Consider using :func:`piesno` to estimate sigma instead if visual inacuracies
-    are apparent in the denoised result.
+    some regions and leave others un-denoised for spatially varying noise
+    profiles. Consider using :func:`piesno` to estimate sigma instead if visual
+    inacuracies are apparent in the denoised result.
```

Comment:
Typo (was there before, but maybe we fix it here?): 

"inacuracies"=> "inaccuracies"